app/models/efile_submission.rb

```diff
@@ -119,10 +119,15 @@ def previously_transmitted_submission
     previous_submission if previous_submission.last_transition_to("transmitted").present?
   end
 
-  def generate_verified_address
+  def generate_verified_address(i = 0)
     return OpenStruct.new(valid?: true) if verified_address.present?
 
-    address_service = StandardizeAddressService.new(intake)
+    address_service = StandardizeAddressService.new(intake, read_timeout: 1500)
```

Comment:
a timeout of 1.5 seconds is totally arbitrary. We set a one second timeout in the application call for this and it seemed ok, so decided to go for a bit longer here.